components/dropdown/README.md

```diff
@@ -123,7 +123,7 @@ To make your `d2l-dropdown-button-subtle` accessible, use the following properti
 ## Opener: Context Menu [d2l-dropdown-context-menu]
 `d2l-dropdown-context-menu` is a simple/minimal opener for dropdown content (`d2l-dropdown-content`, `d2l-dropdown-menu` or `d2l-dropdown-tabs`).
 
-<!-- docs: demo code properties name:d2l-dropdown-context-menu align:flex-start autoOpen:true autoSize:false size:medium -->
+<!-- docs: demo code properties name:d2l-dropdown-context-menu sandboxTitle:'Dropdown Context Menu' align:flex-start autoOpen:true autoSize:false size:medium -->
```

Comment:
We likely want to be consistent in the titles (e.g., have the openers all have the "Dropdown with X opener")